packages/components/psammead-story-promo-list/testHelpers/fixtureData.js

```diff
@@ -0,0 +1,60 @@
+const storyPromoData = [
+  {
+    image: {
+      alt: 'Diosdado Cabello',
+      src:
+        'https://ichef.bbci.co.uk/news/660/cpsprodpb/AF0F/production/_106651844_cabello.jpg',
+    },
+    info: {
+      headline:
+        'Gobierno de Venezuela condena enérgicamente un "intento de golpe" de Juan Guaidó y Leopoldo López',
+      summary:
+        'El liberado Leopoldo López y Juan Guaidó llaman a la "fase final" contra el gobierno de Nicolás Maduro',
+      time: '2019-03-01T14:00+00:00',
```

Comment:
You dont appear to be using the time value here.
Can you add a key for the date format `12 March 2019` and use both please?